SPECS/usrsctp/usrsctp.spec

```diff
@@ -0,0 +1,67 @@
+Summary:        A portable SCTP userland stack
+Name:           usrsctp
+Version:        0.9.5.0
+Release:        1%{?dist}
+License:        LGPLv2+ AND CC0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/sctplab/usrsctp
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz
```

Comment:
This filename needs to be changed.  Its not in a format acceptable for mariner. 
It should always be "%{name}-%{version}.tar.gz"

When you upload the source tarball, there is an option to set the filename.  Please do it as:
usrctp-0.9.5.0.tar.gz

Reply:
Source is uploaded

Reply:
Acknowledged.